querybook/server/lib/ai_assistant/assistants/openai_assistant.py

```diff
@@ -45,24 +43,25 @@ def title_generation_prompt_template(self) -> ChatPromptTemplate:
             "===Example response\n"
             "This is a title\n"
         )
-        human_message_prompt = HumanMessagePromptTemplate.from_template(human_template)
-        return ChatPromptTemplate.from_messages(
-            [system_message_prompt, human_message_prompt]
-        )
+        messages = {
+            {"role": "system", "content": system_message_prompt},
+            {"role": "user", "content": human_message_prompt}
+        }
+        return messages
```

Comment:
follow the same format as chatGPT completions:

https://platform.openai.com/docs/guides/gpt/chat-completions-api